### ISSUE TITLE
🐛 Bug Report: attribute stuck on processing state

### ISSUE THREAD

**Issue by kirstenintech** [appwrite/appwrite#8595]
### 👟 Reproduction steps

1. On August 28, I created a new attribute for my collection during a period when the services were unstable.
2. The attribute creation process initiated but remained in a "processing" status.
3. After the services returned to normal, I attempted to delete the attribute but was unable to do so.
4. I then tried to update the data in table, but encountered an error message stating "Invalid document structure" related to the attribute that is still in the "processing" status.

### 👍 Expected behavior

1. I should be able to delete the attribute or update the table without any errors.

### 👎 Actual Behavior

1. The attribute is stuck in a "processing" status and cannot be deleted.
2. Attempting to update the table results in an "Invalid document structure" error related to the processing attribute.

![Screenshot 2024-08-29 212307](https://github.com/user-attachments/assets/74aef5a4-ad33-4f6d-a01f-56144bddd8f7)
![image](https://github.com/user-attachments/assets/5214b584-a0a8-4663-a3c7-8fa88beb9a3b)

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by tessamero** [comment]
@kirstenintech  You can delete it on your end for now and create another attribute with a different name. The old attribute will show as “Stuck” but you can ignore that for now. We will add a feature soon that will allow you to delete “stuck” attributes. If you really need it deleted completely and cannot wait, you can ping one of the cloud engineers (Christy) and he can help. This is on our radar to implement soon :)

**Comment by kirstenintech** [comment]
Hi @tessamero I created another attribute, but because the original one is "stuck" I can't load any info on the collection.

**Comment by NobodyExceptHim** [comment]
Yup, same here. I've created a SQL to scrape everything from the db that contains anything with my attributes name in it and it's still "processing" in the UI. I'm on self hosted and i would be grateful for some info on where does the UI keep it's data about scheduled tasks as to where can i repair it. Right now i cannot update/read it from the UI because it tells me that the selected attribute does not exist.

**Comment by 3rock618** [comment]
I'm having the same issue. It was caused when I tried to update the name of an attribute. Now when I refresh the page, the new name does not show there, but I cannot edit it either because:

![Image](https://github.com/user-attachments/assets/cb5b1b01-6355-4d74-bd5d-517e4144a26a)

Trying to create a document fails:
![Image](https://github.com/user-attachments/assets/7c683403-c221-4509-b9f7-fe7a82578640)

Initially I changed 3 different attributes. After 15 mins or so it appears one of the attributes finally updated, but the other two are still stuck.

**Comment by stnguyen90** [comment]
@kirstenintech, @NobodyExceptHim, @3rock618, these types of issues require more details and deeper investigation. If you're on the Appwrite Pro plan, please create a support ticket via the Appwrite Console. Otherwise, please create a support post on [Discord](https://appwrite.io/discord/). In your ticket/post, please makes sure to include your project ID, database ID, collection ID, and error.
